templates/emails/plain/failed-refund-admin.php

```diff
@@ -0,0 +1,42 @@
+<?php
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+echo esc_html( $email_heading ) . "\n\n";
+
+printf(
+	// translators: 1) an order number, 2) the customer's full name, 3) the reason for the failure.
+	esc_html_x(
+		'The refund request for order %1$s from %2$s has failed. Reason: %3$s. The customer was also notified.',
+		'In admin refund failed email',
```

Comment:
Should we make the "customer was also notified" conditional on whether that email is hooked in at runtime? It feels like we could get this wrong if merchants or admins disable that email.